src/utils/version.ts

```diff
@@ -89,6 +89,8 @@ export function versionGreaterOrEqualThan(v1: SemVer, v2: SemVer): boolean {
     return true;
   } else if (v1.pre && !v2.pre) {
     return false;
+  } else if (v1.pre && v2.pre && v1.pre === v2.pre) {
+    return v1.build === v2.build;
```

Comment:
I don't think this is correct for this function -- it's meant to implement `>=` so only checking strict equality isn't quite right